src/services/codefixes/references.ts

```diff
@@ -0,0 +1,3 @@
+///<reference path='..\services.ts' />

```

Comment:
we stoped using `references` for a while. they should be picked up by the tsconfig.json in this folder any ways. so please remove this.